modules/ROOT/pages/dataweave-cookbook.adoc

```diff
@@ -4,11 +4,14 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: studio, anypoint, transform, transformer, format, aggregate, rename, split, filter convert, xml, json, csv, pojo, java object, metadata, dataweave, data weave, datamapper, dwl, dfl, dw, output structure, input structure, map, mapping, examples
 
```

Comment:
Edit instructions

Scope: Intro paragraph, subsection titles and subsection intro. You can omit the xref descriptions.

Note: We are reorganizing the DW documentation, as DW will have its own level in the ToC, it will not be embedded in the Mule Runtime docs anymore. Therefore for the DW Examples landing page, we want to have 2 sections that categorize the following:

- DW standalone examples
- Mule application examples that use DW

We would like your input on how to name these sections so they are clear for the user, you will see our comments in the PR.